backend/src/routes/user.js

```diff
@@ -0,0 +1,33 @@
+const userController = require('../controllers/user');
+const userValidators = require('../validators/user');
+
+const routes = [
+    {
+        method: 'GET',
+        url: '/api/user',
+        handler: userController.getAll
+    },
+    {
+        method: 'GET',
+        url: '/api/user/:id',
+        schema: userValidators.getUserValidator,
+        handler: userController.getUser
+    },
+    {
+        method: 'POST',
```

Comment:
so just wondering why the post, put and delete didn't get a schema?

Reply:
Fair question. For this PR, since it was an exploratory one, I just set up what I felt was necessary to demonstrate the technology. I feel like if we're going forward with this then we can add the other validators when we do WD-3732 or WD-3745